### ISSUE TITLE
[Feature Request] use SQL Model to integrate Alchemy and Pydantic

### ISSUE THREAD

**Issue by ELC** [PythonNest/PyNest#44]

**Comment by ItayTheDar** [comment]
Hi @ELC! Thanks for the Feature Request, I really wish that our core team would be able to work on that asap.

**Comment by ELC** [comment]
I can contribute with a PR hut saw that there is a big PR waiting to be merged, any plans to merge that soon?

**Comment by itay-dar-lmnd** [comment]
yes, the PR that introduces the module decorator is about to be pushed in the upcoming week. So I will try to rush this a little bit for you to open the SQLModel Pr

**Comment by ItayTheDar** [comment]
yes, the PR that introduces the module decorator is about to be pushed in the upcoming week. So I will try to rush this a little bit for you to open the SQLModel Pr

**Comment by thongtr-dev** [comment]
Hi, any update on this guys?
